packages/matter.js/src/device/Endpoint.ts

```diff
@@ -53,9 +62,22 @@ export class Endpoint {
         this.addClusterServer(this.descriptorCluster);
         this.setDeviceTypes(deviceTypes);
 
-        if (endpointId !== undefined) {
-            this.id = endpointId;
+        if (options.endpointId !== undefined) {
+            this.id = options.endpointId;
         }
+        if (options.uniqueId !== undefined) {
+            this.uniqueId = options.uniqueId;
+        }
+    }
+
+    setStructureChangedCallback(callback: () => void) {
```

Comment:
Do we have a standard way of handling events?  E.g. `onStructureChanged()` or `structureChanged.on`?  Or is this just used internally?

Reply:
Currently we use callback style ... also eg to store Fabrics or also to inform upper layers that eg a connection is closed to cleanup. So right now no events ....